gallery/src/components/NxDropdown/NxDropdownNavigationExample.tsx

```diff
@@ -18,7 +18,6 @@ function NxDropdownNavigationExample() {
                 className="nx-dropdown--navigation"
                 isOpen={isOpen}
                 onToggleCollapse={onToggleCollapse}>
-      <h4 className="nx-dropdown-title">Menu Title</h4>
```

Comment:
Are we no longer supporting menu titles here?

Reply:
> Are we no longer supporting menu titles here?

According to @dturner32 we are not, the ones that exist in IQ will be custom to IQ until they get refactored out I guess.